guides/release/getting-started/quick-start.md

```diff
@@ -390,7 +390,7 @@ Now that your app is deployed, what should you do next?
 ### Advance to the next level
 
 There is an official, free tutorial here in the Guides that delves deeper into some of the features you used today.
-[Give it a try!](./tutorial)
+[Give it a try!](./../../tutorial/part-1)
```

Comment:
```suggestion
[Give it a try!](../../tutorial/part-1/)
```

I updated this PR with the contents of master and tests are failing for a new reason. Try this to see if it gets past the link checker ^^^